utils/src/main/java/org/corfudb/utils/lock/LockClient.java

```diff
@@ -48,9 +48,13 @@
     // single threaded scheduler to monitor locks
     private final ScheduledExecutorService lockMonitorScheduler;
 
+    private final ScheduledExecutorService taskScheduler;
+
+    private final ExecutorService lockListenerExecutor;
+
     // duration between monitoring runs
     @Setter
-    private static int DurationBetweenLockMonitorRuns = 60;
+    private static int DurationBetweenLockMonitorRuns = 30;
```

Comment:
What is the original value? The monitoring can be smaller so that when the lease expires, the other waiters can get hold of the lease quickly.

Reply:
The original lease value is 60seconds, to 10 maybe?